config/routes.rb

```diff
@@ -2,6 +2,14 @@
 
 Rails.application.routes.draw do
   devise_for :user, controllers: { registrations: "users/registrations" }
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
   root to: "root#index"
+
+  get "/articles/new", to: "article#new", as: "new_article"
+  post "/articles/new", to: "article#new", as: "articles"
+  get "/articles/:id/edit", to: "article#edit", as: "edit_article"
+  patch "/articles/:id/edit", to: "article#edit"
+  delete "/articles/:id", to: "article#remove", as: "remove_article"
+  get "/articles", to: "article#index", as: "self_articles_list"
+  get "/articles/:id", to: "article#index", as: "articles_list"
```

Comment:
This is actually `show` method
```suggestion
  get "/articles/:id", to: "article#show", as: "article"
```